### PR TITLE
Improve the thin-out script

### PR DESCRIPTION
**Pull request by gwicke**
- automatically step over problematic token ranges
- consistently report the current token range

### CODE REVIEW

File: maintenance/thin_out_key_rev_value_data.js [hunk 1]
```diff
@@ -183,12 +194,21 @@ function nextPage(pageState, retryDelay) {
     })
     .catch(function(err) {
         retryDelay = retryDelay || 1; // ms
-        // Retry at least roughly once per 30 minutes
-        if (retryDelay < 900 * 1000) {
+        if (retryDelay < 20 * 1000) {
```
Comment (d00rman) [inline maintenance/thin_out_key_rev_value_data.js]: Put 20000 right away ?

**Comment by d00rman** [comment]
LGTM, but jshint complains *'startOffset' was used before it was defined. at maintenance/thin_out_key_rev_value_data.js:149* so its declaration should be moved up before `processRow()`

File: maintenance/thin_out_key_rev_value_data.js (hunk 1 above)
Comment (gwicke) [inline maintenance/thin_out_key_rev_value_data.js]: Maybe.. it's harder to read when the # of seconds is larger. Should be optimized away anyway.

**Pull request merged**